Objects/StrategyCardAutomator.ttslua

```diff
@@ -0,0 +1,1648 @@
+				BroadcastToColor("Jae Mir Kan is available to you. No Token has been used.", color)
+				return true
+			end
+		elseif (faction.name == "The Yssaril Tribes") and (name == "Ssruu") and (not object.is_face_down) then
+			SsruuActive = true
+		end
+
+		if (name == "Scepter of Emelpar") and (not object.is_face_down) then
+			local zoneColor = _zoneHelper.zoneFromPosition(object.getPosition())
+			if zoneColor == color then
+				_safeBroadcastToColor("Scepter of Emelpar is available to you. No Token has been used.", color, color)
+				return true
+			end
+
+		end
+	end
+
+	if (SsruuActive and JaeMirKanActive) then
+		_safeBroadcastToColor("Ssruu is available to you. No Token has been used.", color, color)
+		return true
+	end
+
+
+	local commandTokensBag = nameToCommandTokensBag[faction.tokenName]
+	if not commandTokensBag then
+		broadcastToAll('Consume Strategy tokens: missing ' .. faction.tokenName .. ' Command Tokens Bag', 'Red')
+		return
+	end
+
+	local commandSheet = getObjectFromGUID(faction.commandSheetGuid)
+	if not commandSheet then
+		broadcastToAll('Consume Strategy tokens: missing ' .. color .. ' Command Sheet', 'Red')
+		return
+	end
+
+	local hitlist = Physics.cast({
+			origin = commandSheet.positionToWorld({ x = -2, y = -1, z=1.5 }),
+			direction = {0, 1, 0},
+			type = 3,
+			size = {5, 1, 5},
+			maxdistance = 0,
+			debug = false
+		})
+	for _, v in pairs(hitlist) do
+		local name = v.hit_object.getName()
+		if name == string.match(name, ".*Command Token") then
+			local rot = v.hit_object.getRotation()['z']
+			-- rot ~ 0 is strategy token. May be off by a little.
+			if (rot < 90) or (rot > 270) then
+				commandTokensBag.putObject(v.hit_object)
+				_safeBroadcastToColor("A token has been removed from your strategy pool.", color, color)
+				return true
+			end
+		end
+	end
+	return false
+end
+
+-------------------------------- Politics --------------------------------------
+
+function _moveSpeakerToken(color)
+	local faction = _factionHelper.fromColor(color)
+	if not faction then
+		broadcastToAll('Assign speaker token: no faction selected for ' .. color, 'Red')
+		return
+	end
+
+	local speakerToken = nil
+	for _, object in ipairs(getAllObjects()) do
+		local name = object.getName()
+		local guid = object.getGUID()
+		if name == "Speaker Token" then
+			speakerToken = object
+			break
+		end
+	end
+
+
+	local commandSheet = getObjectFromGUID(faction.commandSheetGuid)
+	if not commandSheet then
+		broadcastToAll('Assign speaker token: missing ' .. color .. ' Command Sheet', 'Red')
+		return
+	end
+	local rot = commandSheet.getRotation()
+	zone = _zoneHelper.zoneAttributes(color)
+
+	local loc = _getSpeakerLoc(zone)
+
+	speakerToken.setPositionSmooth( loc, false, true )
+	speakerToken.setRotation(rot)
+
+	broadcastToAll(color .. " has been assigned the new speaker.", color)
+	return true
+end
+
+-- Get corner of zone closest to the system.
+-- also tested for 7 and 8 players
+function _getSpeakerLoc(zone)
```

Comment:
Interesting.  might be a little simpler to position it X units "north" of the faction sheet but I'm ok with getting zone borders.